src/enclave/Enclave/Enclave.cpp

```diff
@@ -22,9 +31,9 @@
 void ecall_encrypt(uint8_t *plaintext, uint32_t plaintext_length,
                    uint8_t *ciphertext, uint32_t cipher_length) {
   // Guard against encrypting or overwriting enclave memory
-  assert(sgx_is_outside_enclave(plaintext, plaintext_length) == 1);
-  assert(sgx_is_outside_enclave(ciphertext, cipher_length) == 1);
-  sgx_lfence();
```

Comment:
Is it safe to remove these lines? If so, we can also remove the comment above.

Reply:
Updated with the OE version.